.buildkite/ftr_search_stateful_configs.yml

```diff
@@ -1,14 +1,16 @@
 disabled:
   # Base config files, only necessary to inform config finding script
   - x-pack/solutions/search/test/api_integration/config.ts
+  - x-pack/platform/test/onechat_api_integration/eval/config.stateful.ts
```

Comment:
the test is listed under disabled, which means that CI will not run it, but you can still run it manualy